webpack/containers/Application.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
+
+import Nav from '../scenes/Nav';
+import Content from './ContentPortal';
+
+const ce = React.createElement;
+
+export default () => {
+  const NavPortal = ReactDOM.createPortal(
+    <Nav />,
+    document.getElementById('lab_features_menu-secondary')
```

Comment:
is there any chance those elements might not be on the page?

Reply:
I think they will be here as long as the `experimental_ui` flag is on. If it's not on the `js` won't be loaded anyway.